portfolio/src/main/webapp/youtube.html

```diff
@@ -4,5 +4,14 @@
     <meta charset="UTF-8">
     <title>Youtube</title>
     <link rel="stylesheet" href="youtube-style.css">
+    <script src="youtube_script.js"></script>
   </head>
+  <body>
+      <form action="/data" method="POST">
+        <p>What's a youtube video idea you have for me?</p>
```

Comment:
Instead of a <p> tag, let's make this a <label>. This is recommended for accessibility purposes: https://www.w3schools.com/tags/tag_textarea.asp.

What do I mean by accessibility purposes? If a user was filling out this form using a screen reader, the screen reader wouldn't know that the <p> text was related to your textarea input. By adding a <label for="whats a youtube video idea you have for me"> and then in the textarea adding an id property that corresponds to this, the screen reader will be able to read out that this textarea is related to the 'whats a youtube video idea you have for me'.

Reply:
You don't have to make your whole website completely accessible for this project, but it's really good to start thinking in this way. We always want to make sure we're building an application that can be accessible for everyone #BuildForEveryone :)